apps/web/.agents/skills/prisma-driver-adapter-implementation/SKILL.md

```
@@ -0,0 +1,638 @@
+metadata:
+  author: Tyler Benfield
+  version: "7.0.0"
+---
+
+# Prisma 7 Driver Adapter Implementation Guide
+
+This skill provides everything needed to implement a Prisma ORM v7 driver adapter for any database.
+
+## Architecture Overview
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)